src/main.ts

```diff
@@ -100,6 +110,177 @@ export class Minifier {
     return path.join(destination, subFilePath);
   }
 
+  isExternal(symbol: ts.Symbol): boolean {
+    // TODO: figure out how to deal with undefined symbols
+    // (ie: in case of string literal, or something like true.toString())
```

Comment:
I guess you are trying to say that TS will give you symbol undefined for the example `true.toString()`, make that statement more explicit.